util_test.go

```diff
@@ -43,22 +43,3 @@ func TestExtractIPID(t *testing.T) {
 		t.Fatalf("Expected IP ID %d but got %d.", expectedIPID, ipID)
 	}
 }
-
-func TestExtractTTL(t *testing.T) {
```

Comment:
No `TestExtractIPID`?

Reply:
Not sure I follow. `TestExtractIPID` is [here](https://github.com/brave/zerotrace/blob/aac5e581c3ef92c4f3938d634a8605e8533255d8/util_test.go#L28-L45).

Reply:
Sorry, missed it in the diff.